src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+/* #region Stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
you can use 
@for $index from 1 through 5 {
    &--#{$index} > &__star:nth-child(-n + #{$index}) {
      background: url(../src/images/star-active.svg) no-repeat center;
    }
  }